rest-service/manager_rest/dsl_functions.py

```diff
@@ -246,6 +311,41 @@ def check_vault_connection(url, token, path):
     return vault_status
 
 
+def check_cloudify_connection(**connection_parameters):
+    status = {
+        'status': True,
+        'message': '',
+    }
+
+    if not connection_parameters.get('protocol'):
+        connection_parameters['protocol'] = 'https'
+
+    if cert := connection_parameters.get('cert'):
+        temp_cert_file = _get_cloudify_cert_temp_file(cert)
+        connection_parameters['cert'] = temp_cert_file.name
+
+    client = CloudifyClient(
+        **connection_parameters,
+    )
+
+    try:
+        status_response = client.manager.get_status()
+        status_response.raise_for_status()
+    except requests.exceptions.RequestException as e:
+        status['status'] = False
+        status['message'] = e
```

Comment:
```suggestion
        status['message'] = str(e)
```
alas, an exception object won't be able to be jsonified